horovod/run/run.py

```diff
@@ -265,6 +268,20 @@ def _driver_fn(all_host_names, local_host_names, settings):
         driver.shutdown()
 
 
+def _get_driver_ip(common_intfs):
+    iface = list(common_intfs)[0]
+    server_ip = None
```

Comment:
The method is called `_get_driver_ip` while the variable name is `server_ip` here. I think we should just call the method `get_local_ip` since that is what it does.

Reply:
This is not only local ip.
A machine could have multiple local ip, but not all local ip could be used here.
This method will get such ip on driver machine, which could make sure all worker could connect to this ip. So call it "driver ip" is more accurate.